src/components/10-atoms/checkbox/index.scss

```diff
@@ -172,10 +172,10 @@ axa-checkbox {
   .a-checkbox__children-inline,
   .a-checkbox__children-inline * {
       display: inline;
-      @include typo(primary, small-1);
+      @include text(t4);
 
-      @include breakpoint($mediaquery-sm-up) {
-         @include typo(primary, medium);
+      @include breakpoint($mediaquery-md-up) {
```

Comment:
You changed the behaviour from xs to sm. Because the breakpoint also changed or a mistake?

Reply:
done